src/execution/vm/bytecode_generator.cpp

```diff
@@ -2128,6 +2133,22 @@ void BytecodeGenerator::VisitBuiltinStringCall(ast::CallExpr *call, ast::Builtin
   LocalVar exec_ctx = VisitExpressionForRValue(call->Arguments()[0]);
   LocalVar ret = GetExecutionResult()->GetOrCreateDestination(call->GetType());
   switch (builtin) {
+    case ast::Builtin::Chr: {
+      // input_string here is a integer type number
+      LocalVar input_string = VisitExpressionForRValue(call->Arguments()[1]);
```

Comment:
So, all of these should be `VisitExpressionForLValue()`. But, there are many of them across multiple functions. Leave as is and fix in another PR.

Reply:
TBH I don't have a great grasp on LValue vs RValue in C++. Maybe we should open up an issue so it doesn't fall through the cracks.